core/protocol/wolkabout/WolkaboutProtocol.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2021 WolkAbout Technology s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef WOLKABOUTCORE_WOLKABOUTPROTOCOL_H
+#define WOLKABOUTCORE_WOLKABOUTPROTOCOL_H
+
+#include "core/Types.h"
+#include "core/model/Message.h"
+
+#include <memory>
+
+namespace wolkabout
+{
+const std::string CHANNEL_DELIMITER = "/";
```

Comment:
These should not be in a header

Reply:
They need to be in the `WolkaboutProtocol`, because they're shared across all the `Wolkabout*Protocol` classes. I will put them at least as static members of the class.